src/Themes/VuetifyTheme/Migrations.cs

```diff
@@ -1274,5 +1281,86 @@ private void CreateFip()
                 )
             );
         }
+
+        private void VListTitle() {
+            _contentDefinitionManager.AlterTypeDefinition("VListTitle", type => type
+                .DisplayedAs("VListTitle")
+                .WithPart("VListTitle", part => part
+                    .WithPosition("0")
+                )
+            );
+
+            _contentDefinitionManager.AlterPartDefinition("VListTitle", part => part
+                .WithField("Title", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("Title")
+                    .WithPosition("0")
+                )
+            );
+        }
+        private void VListItem() {
+            _contentDefinitionManager.AlterTypeDefinition("VListItem", type => type
+                .DisplayedAs("VListItem")
+                .WithPart("VListItem", part => part
+                    .WithPosition("0")
+                )
+            );
+
+            _contentDefinitionManager.AlterPartDefinition("VListItem", part => part
+                .WithField("ItemTitle", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("ItemTitle")
+                    .WithPosition("0")
+                )
+                .WithField("ItemSubTitle", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("ItemSubTitle")
+                    .WithPosition("3")
+                )
+                .WithField("ItemText", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("ItemText")
+                    .WithPosition("2")
+                )
+                .WithField("IconName", field => field
+                    .OfType("TextField")
+                    .WithDisplayName("IconName")
+                    .WithPosition("1")
+                )
+            );
+        }
+
+        private void VList() {
+            _contentDefinitionManager.AlterTypeDefinition("VList", type => type
+                .DisplayedAs("VList")
+                .Creatable()
+                .Listable()
+                .Draftable()
+                .Versionable()
+                .Securable()
+                .Stereotype("Widget")
+                .WithPart("VList", part => part
+                    .WithPosition("0")
+                )
+                .WithPart("Items", part => part
+                    .WithDisplayName("Items")
```

Comment:
I think this is missing the OfType(Bag) to set the type of this collection. Maybe missing a name as well.